### ISSUE TITLE
Arch users, `--dns=` required for Firefox to connect to internet for wired interfaces?

### ISSUE THREAD

**Issue by rieje** [netblue30/firejail#1111]
On my Arch desktop system via ethernet interface, without using `--dns=`, Firefox cannot connect to the internet. According to the docs, firejail should use the default dns addressed used by the system if you don't specify `--dns=`, so Firefox should be able to connect to the internet. 

On my Arch laptop system via wireless interface, Firefox can connect to the internet without `--dns=` as expected.

I was wondering if this is a bug on either Arch or firejail's end and if not, how can I get firejail to use the default dns to connect to the internet? It would be pain to specify `--dns=` for all my applications that I wish to connect to the internet with, especially if I want to change the dns address for them in the future.

**Comment by xenopeek** [comment]
I don't have any such issues. This must be a problem on your end. Have you investigated from within the sandbox? E.g. start Firefox with `firejail --name=firefox firefox` and then on a second terminal join the sandbox with `firejail --join=firefox`. Do your tests from there. Like compare DNS configuration in `nmcli` output. Test `dig google.com` and test `dig @8.8.8.8 google.com` and so on.

**Comment by rieje** [comment]
@xenopeek how did you enable wired network on Arch? All I did was:

    systemctl enable --now systemd-networkd.serivce
    systemctl enable --now systemd-resolved.service

**Comment by rieje** [comment]
Thanks for the info--that's what I used to do for wired internet but I think I used my 2 services because it was a more recent recommendation. I disabled those and enable/started the dhcpcd service instead and now the sandboxing works. Going to see what the differences between the services are, but in any case, this appears to be an Arch-related issue (or at least my misunderstanding of services on Arch).
